src/errno.cr

```diff
@@ -43,7 +43,7 @@ enum Errno
   def self.value : self
     {% if flag?(:linux) || flag?(:dragonfly) %}
       Errno.new LibC.__errno_location.value
-    {% elsif flag?(:darwin) || flag?(:freebsd) || flag?(:openbsd) %}
+    {% elsif flag?(:darwin) || flag?(:bsd) %}
```

Comment:
Are we sure we want to do this? Both are not equivalent. Can it override the previous line?

Reply:
This change is fine. It says: do this for all bsd targets. If some bsd target needs different handling, it needs to be handled before that (like the dragonfly branch).

Reply:
@straight-shoota ok, thanks for the answer :+1: